### ISSUE TITLE
HttpFoundation Response object is not available when using WebDriver

### ISSUE THREAD

**Issue by youstra89** [symfony/panther#568]
There is an issue when using $this->assertResponseIsSuccessful() or $this->assertResponseRedirects() in test with Symfony Panther. Follow this link for more details please [https://stackoverflow.com/questions/73670467/symfony-panther-httpfoundation-response-object-is-not-available-when-using-webd](https://stackoverflow.com/questions/73670467/symfony-panther-httpfoundation-response-object-is-not-available-when-using-webd)

**Comment by dunglas** [comment]
Yes it's expected and documented. The WebDriver protocol doesn't give access to the underlying HTTP response.
